app/femr/ui/views/triage/index.scala.html

```diff
@@ -263,6 +250,19 @@ <h2 class="text-center">Check In - Triage</h2>
                     <input type="hidden" name="isDiabetesScreenPerformed" value="false" />
                 </div>
                 }
+        <div class="submitResetWrap">
+            <button type="submit" id="triageSubmitBtn" class="fButton fRedButton fSubmitButton pull-right">
+                Submit Patient</button>
+            <button type="button" class="fButton fOtherButton newPatientBtn pull-left">Reset Fields</button>
+            @if(viewModel.getPatient.getId != 0) {
+                @helper.form(action = SearchController.handleSearch("history")) {
+                    <input type="text" class="hidden" name="patientSearchQuery" value="@viewModel.getPatient.getId"/>
+                    <button type="submit" id="historyButton" class="fButton fOtherButton pull-left">Patient History</button>
+                }
+            }
+
```

Comment:
This code works pretty well, but a form within a form isn't valid HTML. My thoughts are that this button does not need to be a form element. We could potentially replace the second form with an anchor tag like so:

```
@if(viewModel.getPatient.getId != 0) {
     <a href="@HistoryController.indexPatientGet(viewModel.getPatient.getId.toString)" class="fButton fOtherButton pull-left">Patient History</a>
}
```

It will just take some minor css to make the anchor tag look like the buttons on the page. It shoud get pretty close using the same classes though.